app/templates/participate/login.hbs

```diff
@@ -7,20 +7,30 @@
         {{/if}}
         <button id="select-language" class="btn btn-default" {{action 'showLanguageSelector'}}>{{t 'login.changeButton'}}</button>
       </h3>
+    {{else}}
+        {{#unless showLanguageSelector}}
+            <button id="select-language" class="btn btn-default" {{action 'showLanguageSelector'}}>{{t 'login.changeButton'}}</button>
+        {{/unless}}
     {{/if}}
 
     {{#if invalidAuth}}
-      {{#bs-modal title="An error occured when logging in." closedAction=(action 'toggleInvalidAuth')}}
+      {{#bs-modal title="An error occured when logging in." closedAction=(action 'toggleAction' 'invalidAuth')}}
         {{#bs-modal-body}}Invalid Study ID or Participant ID.{{/bs-modal-body}}
```

Comment:
I think the error messages can be translated since all of the text on the login page is also being translated (it just defaults to english until a user has chosen a language).